lib/transport/transport.ts

```diff
@@ -33,7 +43,10 @@ export function marshallTransform(jsValue: any, context: transportable.Transport
         if (constructorName !== 'Object') {
             if (typeof jsValue['cid'] === 'function') {
                 return <transportable.Transportable>(jsValue).marshall(context);
-            } else {
+            } else if (_v8TransportHelperVerifiedWhitelist.has(constructorName)) {
+                return {v8SerializedData : v8TransportHelper.serializeValue(jsValue)};
```

Comment:
How about to introduce a new typescript class to hold the v8serializedData?

Reply:
Offline discussed, will keep the current structure because of the execution process of Json.stringify

---
In reply to: [154793085](https://github.com/Microsoft/napajs/pull/137#discussion_r154793085) [](ancestors = 154793085)